cluster/juju/layers/kubernetes-master/actions/get-kubeconfig

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/sbin/charm-env python3
+import os
+from yaml import safe_load as load
+from charmhelpers.core.hookenv import (
+    action_get,
+    action_set,
+    action_fail,
+    action_name
+)
+from charmhelpers.core.templating import render
+from subprocess import check_output
+
+
+def _kubectl(args):
+    """
+    Executes kubectl with args as arguments
+    """
+    snap_bin = os.path.join(os.sep, "snap", "bin")
+    env = os.environ.copy()
+    env["PATH"] = os.pathsep.join([snap_bin, env["PATH"]])
+    cmd = ["kubectl", "--kubeconfig=/home/ubuntu/config"]
+    cmd.extend(args)
+    return check_output(cmd, env=env)
+
+
+def get_kubeconfig():
+    """
+    Read the kubeconfig on the master and return it as JSON
+    """
+    kubeconfig = load(_kubectl(["config", "view", "-o", "yaml", "--raw"]))
+    action_set({"credential": kubeconfig})
```

Comment:
I would prefer this be "kubeconfig" instead of "credential" since that more explicitly describes the output value.

Reply:
Good point! Will change that.